lib/chef/chef_fs/file_system/cookbooks_dir.rb

```diff
@@ -106,7 +106,7 @@ def upload_versioned_cookbook(other, options)
             cookbook_to_upload.freeze_version if options[:freeze]
 
             # Instantiate a new uploader based on the proxy loader
-            uploader = Chef::CookbookUploader.new(cookbook_to_upload, :force => options[:force], :rest => root.chef_rest)
+            uploader = Chef::CookbookUploader.new(cookbook_to_upload, :force => options[:force], :rest => root.rest)
```

Comment:
It should but its breaking tests. I'm gonna leave it as `chef_rest` but we will be deprecating it in 13.